fastlane_core/lib/fastlane_core/itunes_transporter.rb

```diff
@@ -99,7 +104,10 @@ def execute(command, hide_output)
         UI.important("Although errors occurred during execution of iTMSTransporter, it returned success status.")
       end
 
-      exit_status.zero?
+      {
+        successful: exit_status.zero?,
+        output: @all_lines
+      }
```

Comment:
It looks like this is needed in the `provider_ids` function in `ItunesTransporter` to parse out the response to be able to infer the `itc_provider`?

Reply:
That's accurate—the iTMS Transporter response was previously discarded. I like the idea in Josh's comment further down, where this would send the response out through a block instead of changing the return type.